glyphs2fontir/src/erase_open_corners.rs

```diff
@@ -102,22 +102,31 @@ impl<'a> CornerErasureCtx<'a> {
         let Some(maybe_corner) = self.possible_corner(seg_ix) else {
             return false;
         };
+        log::debug!("considering segment starting at {}", maybe_corner.one.end());
         if !maybe_corner.points_are_right_of_line() {
+            log::debug!(
+                "crossing points {} and {} not on same side of line",
+                maybe_corner.point_before_line(),
+                maybe_corner.point_after_line()
+            );
```

Comment:
Optional, I think many of the debugs here might be better set to trace.

Reply:
agree, had chosen debug (and these particular statements) because I wanted our debug output to be the same as python's, so that I could compare the logs and figure out what was going on. trace should be fine for that, though..